hat/assets/js/apps/Iaso/bundle/generators.js

```diff
@@ -0,0 +1,279 @@
+const fs = require('fs');
+const path = require('path');
+const { getAvailableLanguages } = require('./languages.js');
+const { getPluginFolders } = require('./plugins.js');
+
+/** @param {string} rootDir */
+/** @param {string[]} availableLanguages */
+
+const generateCombinedTranslations = rootDir => {
+    const availableLanguages = getAvailableLanguages(rootDir);
+    const combinedTranslationsPath = path.resolve(
+        rootDir,
+        './assets/js/apps/Iaso/bundle/generated/combinedTranslations.js',
+    );
+
+    // Create a combined translations object
+    const combinedTranslations = {};
+
+    availableLanguages.forEach(lang => {
```

Comment:
To be consistent, you should use `reduce` i.o `forEach`. We should update the guideline and/or make this helper thingy